frontend/src/routes/Credentials/CredentialsForm.tsx

```diff
@@ -84,14 +84,16 @@ const providerGroup: Record<string, string> = {
     [Provider.hybrid]: ProviderGroup.CentrallyManaged,
 }
 
-export default function CredentialsFormPage({ match }: RouteComponentProps<{ namespace: string; name: string }>) {
-    const { name, namespace } = match.params
+export default function CredentialsFormPage() {
+    const params = useParams<{ namespace: string; name: string }>()
+    const location = useLocation()
+    const { name, namespace } = params
     const { t } = useTranslation(['credentials', 'common'])
 
     let isEditing = false
     let isViewing = false
     if (name !== undefined) {
-        isEditing = match.path.endsWith(NavigationPath.editCredentials)
+        isEditing = location.pathname.startsWith('/multicloud/credentials/edit')
```

Comment:
Should we hold off committing this change until we get parameterized routes working?

Reply:
they are working... had to change it because of the way we have to access the params now